### DISCUSSION TITLE
Mish Friendly Food 1.0

### DISCUSSION THREAD

**Discussion by mishmanners** [mishmanners/mish-friendly-food#82] [Announcements]
# Mish Friendly Food first ever release 🥳 

[We now have a website](https://mishfriendlyfood.com), which is awesome 🎊 
Phase 1 of the Project is now complete, hence release 1.0.

<img width="1270" alt="Mish Friendly Food Release" src="https://user-images.githubusercontent.com/36594527/199144240-43ad7787-35bd-4eb6-84da-79eed15b749d.PNG">

### Major features
* Add the why by @mishmanners in https://github.com/mishmanners/mish-friendly-food/pull/1
* Create CODE_OF_CONDUCT.md by @mishmanners in https://github.com/mishmanners/mish-friendly-food/pull/3
* Add phases of the project by @mishmanners in https://github.com/mishmanners/mish-friendly-food/pull/2
* Setup Astro by @amykapernick in https://github.com/mishmanners/mish-friendly-food/pull/20
* Add Croatian translations database.json by @MartinaVukanovic in https://github.com/mishmanners/mish-friendly-food/pull/23
* Updated a few Russian translations by @andrekolodochka in https://github.com/mishmanners/mish-friendly-food/pull/25
* Update database.json by @MartinaVukanovic in https://github.com/mishmanners/mish-friendly-food/pull/26
* Added Danish translation by @lklint in https://github.com/mishmanners/mish-friendly-food/pull/31
* Updated Spanish translations by @AndreaGriffiths11 in https://github.com/mishmanners/mish-friendly-food/pull/30
* Updates to Portuguese by @pachicodes in https://github.com/mishmanners/mish-friendly-food/pull/29
* Fix for some Japanese translations by @parroty in https://github.com/mishmanners/mish-friendly-food/pull/27
* Add Logo & Error 404 page images, resolves #24 by @VaishantMakan in https://github.com/mishmanners/mish-friendly-food/pull/32
* Update French translations by @Quenrelia in https://github.com/mishmanners/mish-friendly-food/pull/50
* Added a custom 404 by @lyraddigital in https://github.com/mishmanners/mish-friendly-food/pull/48
* Added Translations Page by @amykapernick in https://github.com/mishmanners/mish-friendly-food/pull/46
* Update Hindi translations by @VaishantMakan in https://github.com/mishmanners/mish-friendly-food/pull/57
* Update Arabic translation 🇱🇧  by @fatenhealy in https://github.com/mishmanners/mish-friendly-food/pull/63

### Minor features
* Tweaked layout styles and added a couple of different page examples by @amykapernick in https://github.com/mishmanners/mish-friendly-food/pull/22
* Fix json formatting by @parroty in https://github.com/mishmanners/mish-friendly-food/pull/28
* Add Feature request template by @mishmanners in https://github.com/mishmanners/mish-friendly-food/pull/36
* Add translation request template by @mishmanners in https://github.com/mishmanners/mish-friendly-food/pull/38
* Add Discussions link by @mishmanners in https://github.com/mishmanners/mish-friendly-food/pull/39
* List of supported languages by @mishmanners in https://github.com/mishmanners/mish-friendly-food/pull/41
* Add favicon manifest and serve images by @glennsarti in https://github.com/mishmanners/mish-friendly-food/pull/42
* Add development container files by @glennsarti in https://github.com/mishmanners/mish-friendly-food/pull/43
* Add a basic CI test by @glennsarti in https://github.com/mishmanners/mish-friendly-food/pull/45
* CSS and SEO improvements by @lyraddigital in https://github.com/mishmanners/mish-friendly-food/pull/53
* Added build and deploy workflow by @amykapernick in https://github.com/mishmanners/mish-friendly-food/pull/54

## New Contributors
* @mishmanners made their first contribution in https://github.com/mishmanners/mish-friendly-food/pull/1
* @amykapernick made their first contribution in https://github.com/mishmanners/mish-friendly-food/pull/20
* @MartinaVukanovic made their first contribution in https://github.com/mishmanners/mish-friendly-food/pull/23
* @andrekolodochka made their first contribution in https://github.com/mishmanners/mish-friendly-food/pull/25
* @lklint made their first contribution in https://github.com/mishmanners/mish-friendly-food/pull/31
* @AndreaGriffiths11 made their first contribution in https://github.com/mishmanners/mish-friendly-food/pull/30
* @pachicodes made their first contribution in https://github.com/mishmanners/mish-friendly-food/pull/29
* @parroty made their first contribution in https://github.com/mishmanners/mish-friendly-food/pull/28
* @VaishantMakan made their first contribution in https://github.com/mishmanners/mish-friendly-food/pull/32
* @glennsarti made their first contribution in https://github.com/mishmanners/mish-friendly-food/pull/42
* @Quenrelia made their first contribution in https://github.com/mishmanners/mish-friendly-food/pull/50
* @lyraddigital made their first contribution in https://github.com/mishmanners/mish-friendly-food/pull/48
* @fatenhealy made their first contribution in https://github.com/mishmanners/mish-friendly-food/pull/63
* @Viswas-Programs made their first contribution in https://github.com/mishmanners/mish-friendly-food/pull/75

**Full Changelog**: https://github.com/mishmanners/mish-friendly-food/commits/1.0

<hr /><em>This discussion was created from the release <a href='https://github.com/mishmanners/mish-friendly-food/releases/tag/1.0'>Mish Friendly Food 1.0</a>.</em>

**Comment by mishmanners** [comment]
Thank you everyone for helping get this project off the ground 🙏  I'm so happy we could get Phase 1 complete by the end of Hacktoberfest. Let's keep working on this and make it easier for people to travel who have dietary issues!

**Reply by glennsarti** [reply]
Release Party! 🎉🎉🥳🥳

**Reply by Viswas-Programs** [reply]
Yeah when is that?
And, thanks for mentioning my name in the contributors list :) I joined github like 2 weeks ago, and I am here!
